src/editorInterface/index.js

```diff
@@ -10,6 +10,14 @@ const {
   getMarkdownScopes,
 } = require('../atomInterface');
 
+let flow;
+const lazyFlow = () => {
+  if (!flow) {
+    flow = require('lodash/fp/flow'); // eslint-disable-line global-require
+  }
+  return flow;
+};
+
```

Comment:
Maybe we will be able to revert this in a future PR if we implement babel-plugin-lodash? Or at least we can clean up a bit by implementing that import-lazy lib?

Reply:
Ya, I think there's a larger story to be tackled of "What's the pattern for lazy loading?".

This mirrors what's being used in `main.js`, but it can become a bit verbose. 
Another PR for `import-lazy` would be interesting.